File: allofmycollectionprojects/Sqlcipher/TestSqlCipher/TestSqlCipher/ViewController.m
//
//  ViewController.m
//  TestSqlCipher
//
//  Created by LyGV on 3/13/13.
//  Copyright (c) 2013 LyGV. All rights reserved.
//

#import "ViewController.h"
#import <SqlCipher/SqlCipher.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
    NSString *p = [[NSBundle mainBundle] pathForResource:@"masterdata.sqlite3" ofType:nil];
    SqlCipher *sql = [SqlCipher databaseWithPath:p andKey:@"21fc123678edfd999aaab4f378b9110d"];
    if ([sql tableExists:@"cards"]) {
        NSLog(@"%@", [sql fetchDataAsArray:@"select * from skill"]);
    }
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: gamec8d46d341bea4fd5bff866a65ff8aea9/2048Noti/2048Noti/ViewController.swift
//
//  ViewController.swift
//  2048Noti
//
//  Created by LyGV on 11/20/14.
//  Copyright (c) 2014 lygv. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    var game:NumberTileGameViewController?
    override func viewDidLoad() {
        super.viewDidLoad()
        game = NumberTileGameViewController(dimension: 4, threshold: 2048)
        game?.view.center = self.view.center;
        view.addSubview(game!.view)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

